ansible/roles/riakcs/configure/defaults/main.yml

```diff
@@ -11,5 +11,6 @@ riak_data_root_leveldb: "{{ riak_data_dir }}/leveldb"
 riak_data_root_bitcask: "{{ riak_data_dir }}/bitcask"
 riak_ring_dir: "{{ riak_data_dir }}/ring"
 riak_nodename: "riak-{{ inventory_hostname_short }}@{{ inventory_hostname }}"
+riak_ring_size: 64
```

Comment:
oh, this is the default, not the new value for ICDS. we already had this discussion: 43a005976e8766fe78208c836212010ece0a1623